xtractmime/_utils.py

```diff
@@ -0,0 +1,252 @@
+from struct import unpack
+from typing import Tuple, Union
+
+from xtractmime import _is_match_mime_pattern
+from xtractmime._patterns import (
+    IMAGE_PATTERNS,
+    AUDIO_VIDEO_PATTERNS,
+    FONT_PATTERNS,
+    ARCHIVE_PATTERNS,
+)
+
+sample_rates = [44100, 48000, 32000]
```

Comment:
These values do not need to be mutable, so a tuple makes more sense. And non-mutable module-level variables are conceptually constants, which should be uppercased.

```suggestion
SAMPLE_RATES = (44100, 48000, 32000)
```